### PR TITLE
Fix openpilot-prebuilt dockerfile

### PR DESCRIPTION
**Pull request by fredyshox**

### CODE REVIEW

File: Dockerfile.openpilot [hunk 1]
```diff
@@ -13,18 +13,19 @@ COPY SConstruct ${OPENPILOT_PATH}
...
 COPY ./cereal ${OPENPILOT_PATH}/cereal
 COPY ./msgq_repo ${OPENPILOT_PATH}/msgq_repo
-COPY ./msgq ${OPENPILOT_PATH}/msgq
 COPY ./panda ${OPENPILOT_PATH}/panda
 COPY ./selfdrive ${OPENPILOT_PATH}/selfdrive
 COPY ./system ${OPENPILOT_PATH}/system
 COPY ./body ${OPENPILOT_PATH}/body
 
+RUN ln -s ${OPENPILOT_PATH}/rednose_repo/rednose ${OPENPILOT_PATH}/rednose
+RUN ln -s ${OPENPILOT_PATH}/msgq_repo/msgq ${OPENPILOT_PATH}/msgq
```
Comment (adeebshihadeh) [inline Dockerfile.openpilot:29]: why doesn't copying the symlinks work

Comment (fredyshox) [reply Dockerfile.openpilot:29]: Looks like `COPY` always resolves the links

Comment (adeebshihadeh) [reply Dockerfile.openpilot:29]: can you just copy openpilot/ then?

Comment (fredyshox) [reply Dockerfile.openpilot:29]: but only when copying individual files, it preserves them when copying a directory containing links

Comment (fredyshox) [reply Dockerfile.openpilot:29]: > can you just copy openpilot/ then?

yeah, `COPY . ${OPENPILOT_PATH}/` will work, assuming that the dir is clean, which should be fine in CI

**Pull request merged**